core/src/main/scala/sttp/tapir/model/ServerRequest.scala

```diff
@@ -10,6 +10,8 @@ import scala.collection.immutable.Seq
 trait ServerRequest extends RequestMetadata {
   def protocol: String
   def connectionInfo: ConnectionInfo
+  /** Override in backend-specific implementation if you want a more efficient implementation and avoid uri parsing overhead */
+  def uriStr: String = uri.copy(scheme = None, authority = None, fragmentSegment = None).toString
```

Comment:
or maybe impls should simple override `showShort`? Since we are doing some customisations here (removing the scheme etc.), which are not necessarily general

Reply:
Right, since `uriStr` will be used only rather only for `showShort` it seems like a good idea to skip it and just override that method :+1: